index.html

```diff
@@ -67,30 +74,55 @@ <h2>
     <h2>
       Model
     </h2>
-    <dl class="definitions" data-sort="" data-cite="vc-data-model">
+
+    <p class="issue" title="Definitions under discussion">
+The goal of the definitions in this section is to re-use or establish
+terminology that is common across a variety of digital credential formats and
+protocols.
+    </p>
```

Comment:
Let's live dangerously... but seriously, we are ok to just define things within the W3C and our own ecosystem. 

```suggestion
```

Reply:
@marcoscaceres, you rebel, you. :)

I was trying to convey to readers, as this document matures,  that the goal is to align w/ multiple ecosystems and not re-define yet another set of terminology that is inconsistent. I'm fine w/ removing this if others want it gone, but do think it has value to convey that we're still working to align terminology and will continue to strive towards that goal.

Looking for more opinions wrt. revising or removing this block.

Reply:
Ok, to keep it, but change it to a "note" instead of an issue.

Reply:
Changed to NOTE in 6dd3f57d6893649d848fda10077e2b840d89dad8.